### PR TITLE
new gps (gpsfake tested sans pps)

### PR DESCRIPTION
**Pull request by jr1221**
## Changes

_Explanation of changes goes here_

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

File: src/net.rs
Comment (caiodasilva2005) [inline src/net.rs:1]: ah this really cool

File: src/halow.rs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+async fn handle_tick(
+    rssi_regex: &Regex,
+    mcs_regex: &Regex,
+    rssi_topic: String,
+    mcs_topic_tx: String,
+    mcs_topic_rx: String,
+) -> Vec<PublishableMessage> {
+    let mut send: Vec<PublishableMessage> = Vec::with_capacity(2);
+    let output = Command::new("cli_app")
```
Comment (caiodasilva2005) [inline src/halow.rs:49]: sorry for lack of understanding here, but how does the cli_app work? so you are just parsing results from "show signal" and then it just outputs all the datat correlated to the rssi?

Comment (jr1221) [reply src/halow.rs:49]: Yup just screen scraping the command.  It is a  proprietary helper written by the Newracom people (halow driver).
